src/Http/Connection.php

```diff
@@ -19,6 +19,9 @@
 use Symplicity\Outlook\Interfaces\Http\ConnectionInterface;
 use Symplicity\Outlook\Interfaces\Http\RequestOptionsInterface;
 use Symplicity\Outlook\Utilities\RequestType;
```

Comment:
this is a public repo. app specific code can't be here. @frets1700 can comment specifically. but this is just wrong as I see this acc. stuff here.